evaluation/README.md

```diff
@@ -47,6 +47,25 @@ Here, we provide a leaderboards for each default task, by executing transformati
 ### Text Classification
 
 
+| Transformation              | roberta-base-SST-2   | bert-base-uncased-QQP   | roberta-large-mnli   | roberta-base-imdb   |
+|:----------------------------|:---------------------|:------------------------|:---------------------|:--------------------|
+| BackTranslation             | 94.0->91.0 (-3.0)    | 92.0->90.0 (-2.0)       | 91.0->87.0 (-4.0)    | 95.0->92.0 (-3.0)   |
+| ButterFingersPerturbation   | 94.0->89.0 (-5.0)    | 92.0->89.0 (-3.0)       | 91.0->88.0 (-3.0)    | 95.0->93.0 (-2.0)   |
+| ChangePersonNamedEntities   | 94.0->94.0 (0.0)     | 92.0->92.0 (0.0)        | 91.0->89.0 (-2.0)    | 95.0->95.0 (0.0)    |
+| CloseHomophonesSwap         | 94.0->91.0 (-3.0)    | 92.0->88.0 (-4.0)       | 91.0->89.0 (-2.0)    | 95.0->96.0 (1.0)    |
```

Comment:
Many of the results in a naturally occurring setting seem interesting-ButterFingersPerturbation, ChangePersonNamedEntities & CloseHomophonesSwap. 

So, are these on the complete SST-2 and IMDB or a subset (p=10 or 20, etc) ?

Reply:
I did p=20%.
It's possible to compute on the full SST-2 dataset though; For IMDB some of these will be too slow...